css/main.scss

```diff
@@ -2673,18 +2670,20 @@ pre {
       flex-wrap: nowrap;
       border-radius: 4px;
       background-color: #f8fafb;
-      padding: 0px 24px;
-      margin-bottom: 48px;
+      padding: 0rem 2rem 0rem 0.5rem;
+      //margin-bottom: 48px;
       margin-top: 16px;
+      justify-content: space-between;
+      box-sizing: border-box;
+      gap:1rem;
+      
 
       fieldset {
         display: flex;
         flex-wrap: nowrap;
         align-items: center;
-        margin-right: 24px;
-        margin: 0 24px 0 0;
         justify-content: flex-start;
-        min-width: calc(60% + 10px);
```

Comment:
I think here aswell, fieldset is a global selector, which means that you will change all the fieldset elements if you change something here. I think it's also better if you could revert these changes and create a new CSS selector that specifies the element you want to change and apply your CSS on there.